KachnaOnline.Business.Data/Repositories/ReservationItemEventRepository.cs

```diff
@@ -37,5 +37,16 @@ public Task Add(ReservationItemEvent entity)
             Set.Add(entity);
             return Task.CompletedTask;
         }
+
+        public async Task<int?> GetAssignee(int itemId)
+        {
+            var entity = await Set
+                .Where(e => e.ReservationItemId == itemId)
+                .Where(e => e.Type == ReservationEventType.Assigned)
+                .OrderBy(e => e.MadeOn)
+                .Take(1)
+                .FirstOrDefaultAsync();
```

Comment:
Is it necessary to take one and then take the first one, that is, the only one? Is `.Take(1)` not redundant here?

Reply:
Fixed.